README.md

```diff
@@ -21,3 +21,22 @@ git checkout development
 
 Good luck, and have a great day!
 
+## Build the docker file
+1. First, build the esmf docker image. You must tag the build with `esmf`
+```
+cd docker
+docker build -f Dockerfile.esmf -t esmf .
```

Comment:
Knowing nothing about docker, I checked out this branch and it failed on the docker build line with a "command not found"

Reply:
@cacraigucar You'll need to install docker "desktop" on your computer first.  Instructions for doing so can be found online here:

https://docs.docker.com/get-docker/

Reply:
Perhaps this could be added to the README.  I naively assumed since I moved to the docker directory, that the "docker" command would somehow be miraculously in there.